lib/shelley/bench/Latency.hs

```diff
@@ -223,12 +226,12 @@ walletApiBench capture ctx = do
         let expInflows' = filter (/=0) expInflows
 
         mapM_ (repeatPostTx wal1 amt batchSize . amtExp) expInflows'
-        pure (wal1, wal2)
+        pure (wal1, wal2, walMA)
 
     nFixtureWalletWithUTxOs n utxoNumber = do
         let utxoExp = replicate utxoNumber minUTxOValue
         wal1 <- fixtureWalletWith @n ctx utxoExp
-        (_, wal2) <- nFixtureWallet n
+        (_, wal2, walMA) <- nFixtureWallet n
```

Comment:
The `walMA` wallet will only have 1 UTxO, unlike `wal1`, which gets utxoNumber.

Reply:
Yes, however significant difference in time is also visible for "Latencies for 2 fixture wallets scenario" category where all wallets (wal1, wal2, walMA) are just regular `fixtureWallets` / `fixtureMultiAssetWallet` e.g.:

```
    Latencies for 2 fixture wallets scenario
       ...
        postTransactionFee  - 546.1 ms
        postTransaction     - 200.8 ms
        postTransTo5Addrs   - 412.2 ms
        postTransactionMA   - 22.8 ms
```

As far as I see both [fixtureWallet](https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core-integration/src/Test/Integration/Framework/DSL.hs#L1334-L1338) and [fixtureMultiAssetWallet](https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core-integration/src/Test/Integration/Framework/DSL.hs#L1213-L1217) are created by `fixtureWalletWithMnemonics` method underneath... 

As far as I see `fixtureMultiAssetWallet` has 3 UTxO:
```
{
  "distribution": {
    "10000000000000000": 0,
    "100000000000000": 0,
    "1000000000000": 0,
    "10000000000": 1,
    "100000000": 0,
    "1000000": 0,
    "10000": 0,
    "100": 0,
    "1000000000": 2,
    "100000000000": 0,
    "10": 0,
    "45000000000000000": 0,
    "1000": 0,
    "10000000000000": 0,
    "1000000000000000": 0,
    "100000": 0,
    "10000000": 0
  },
  "scale": "log10",
  "total": {
    "quantity": 4000000000,
    "unit": "lovelace"
  }
}

```

`fixtureWallet` has 10 UTxO:

```
{
  "distribution": {
    "10000000000000000": 0,
    "100000000000000": 0,
    "1000000000000": 0,
    "10000000000": 0,
    "100000000": 0,
    "1000000": 0,
    "10000": 0,
    "100": 0,
    "1000000000": 0,
    "100000000000": 10,
    "10": 0,
    "45000000000000000": 0,
    "1000": 0,
    "10000000000000": 0,
    "1000000000000000": 0,
    "100000": 0,
    "10000000": 0
  },
  "scale": "log10",
  "total": {
    "quantity": 1000000000000,
    "unit": "lovelace"
  }
}
```

So there is 7 UTxO difference between those wallets. The difference in sending transaction from fixtureWallet (10 UTxO) and fixtureMultiAssetWallet (3 UTxO) is almost 10x, I wasn't expecting such difference, but maybe that is expected...